src/components/chart/YGrid.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+
+import {XYChartContext} from './XYChart';
+
+export interface YGridProps {
+    padding?: number;
+    color?: string;
+}
+
+export const YGrid: React.FunctionComponent<YGridProps> = ({padding = 0, color = 'rgba(0,0,0,0.2)'}) => {

```

Comment:
im not sure if its a good idea to set all default directly in the function arguments. I think it should be better to have an object with all default value. More ez to manage after